miner/oracle.go

```diff
@@ -154,7 +154,7 @@ func (o *Oracle) calcEligibilityProofs(atx *types.ActivationTxHeader, epoch type
 		)
 	}
 
-	logger.With().Info("proposal eligibility calculated",
+	logger.With().Info("tortoise eligibility for an epoch",
```

Comment:
i am not sure tortoise eligibility is better than proposal eligibility

Reply:
reverted